assembly/assembly-ide-war/pom.xml

```diff
@@ -434,7 +434,6 @@
                     <!-- don' remove it we will use it then need to found bug in compiled JS -->
                     <!--style>DETAILED</style-->
                     <logLevel>${gwt.compiler.logLevel}</logLevel>
```

Comment:
Do we need something like this https://github.com/codenvy/codenvy/blob/cbf975ecdff431b1feacd02cd6743fe2dec3e456/assembly/onpremises-ide-compiling-war-ide/pom.xml#L425 here